dvc/tree/oss.py

```diff
@@ -116,6 +117,18 @@ def remove(self, path_info):
         logger.debug(f"Removing oss://{path_info}")
         self.oss_service.delete_object(path_info.path)
 
+    def _upload_fobj(self, fobj, to_info):
+        cursor = 0
+        while True:
+            chunk = fobj.read(self.CHUNK_SIZE)
+            if not chunk:
+                break
+
+            result = self.oss_service.append_object(
+                to_info.path, cursor, chunk
+            )
+            cursor = result.next_position
```

Comment:
The chunk size might once again be too much for oss to handle :( Can we just use `put_object(fobj)`?